# RemoveMapLocal.srv
string map_name
---
string result
